packages/hooks/use-namespace/index.ts

```diff
@@ -62,15 +62,19 @@ export const useNamespace = (block: string) => {
   const cssVar = (object: Record<string, string>) => {
     const styles: Record<string, string> = {}
     for (const key in object) {
-      styles[`--${namespace.value}-${key}`] = object[key]
+      if (object[key]) {
+        styles[`--${namespace.value}-${key}`] = object[key]
+      }
     }
     return styles
   }
   // with block
   const cssVarBlock = (object: Record<string, string>) => {
     const styles: Record<string, string> = {}
     for (const key in object) {
-      styles[`--${namespace.value}-${block}-${key}`] = object[key]
+      if (object[key]) {
```

Comment:
Just noticed a potential issue -
Will it work if the value is number `0` rather than string `0`?

Reply:
The accepted type is string, and passing the number 0 directly will cause a TS error.

Reply:
It will be compiled to be js file finally.

Reply:
The internal use of the component library should not cause problems unless the user calls the function himself.

Reply:
> unless the user calls the function himself.

This is what I'm concerned about. This function has been exported and is public for the users. It may cause unnecessary issues if they use JS rather than TS to develop their projects. We can either fix it or wait for the feedback from the users.